freqtrade/freqai/freqai_util.py

```diff
@@ -0,0 +1,115 @@
+"""
```

Comment:
What's the difference between `freqtrade.freqai.utils` and `freqtrade.freqai.freqai_utils` ? (both files are utils and contain some utilities ...)

Reply:
I see the issue. Can we move the data_kitchen specific functions directly into `data_kitchen.py`?

Reply:
Move `plot_feature_importance` to `data_kitchen.py` right? and remove this import `from freqtrade.freqai.data_kitchen import FreqaiDataKitchen`? I will modify this to keep just one util file.

Reply:
Done!